### PR TITLE
[JOLINT-70]  co founders section

### PR DESCRIPTION
**Pull request by xxrobone**

### CODE REVIEW

File: app/(info-page)/_components/CoFounders.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  },
+]
+
+const CoFoundersSection = () => {
+  return (
+    <section className='pt-20 md:pt-28'>
+      <h3 className='font-heading text-2xl font-bold text-neutral-900 md:text-4xl'>
+        Co-Founders
+      </h3>
+      <div className='flex flex-col items-center justify-evenly gap-12 md:gap-20'></div>
```
Comment (ell-ska) [inline app/(info-page)/_components/CoFounders.tsx:23]: items-center and justify-evenly are unnecessary? check it they make a difference but i dont think they will

File: app/(info-page)/about/page.tsx [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import OurMissionSection from "@/app/(info-page)/_components/OurMissionSection"
+import CoFoundersSection from "../_components/CoFounders"
```
Comment (ell-ska) [inline app/(info-page)/about/page.tsx:2]: use @ for imports

File: public/faheem.png
Comment (ell-ska) [inline public/faheem.png:1]: this image should be flipped, re export it from figma

**Review by ell-ska — CHANGES_REQUESTED** [review]

File: app/(info-page)/_components/CoFounders.tsx (hunk 1 above)
Comment (xxrobone) [reply app/(info-page)/_components/CoFounders.tsx]: removed them

File: app/(info-page)/about/page.tsx (hunk 2 above)
Comment (xxrobone) [reply app/(info-page)/about/page.tsx]: changed import

File: public/faheem.png
Comment (xxrobone) [reply public/faheem.png:1]: changed image

**Comment by idamakela** [comment]
### Dev testing approved!

**Pull request merged**